src/test/K6/src/tests/platform/authorization/delegations/inheritancev2.js

```diff
@@ -73,12 +78,19 @@ export function setup() {
 
   var aspxauthCookie3 = setUpData.authenticateUser(user3Name, user3Pwd);
   var altinnStudioRuntimeCookie3 = setUpData.getAltinnStudioRuntimeToken(aspxauthCookie3);
-  var userData3 = setUpData.getUserData(altinnStudioRuntimeCookie3, appOwner, appName);
+  var userData3 = setUpData.getUserData(altinnStudioRuntimeCookie3, appOwner, appName, user3OrgNo);
 
   var ecUserData;
-  if(minimumSBLVersion(22, 5)) {
+  if(minimumSBLVersion(22, 5) && environment != 'tt02') {
     ecUserData = setUpData.authenticateECUser(ecUserName, ecUserPwd, ecUserOrgNo);
   }
+  else {
+    ecUserData = {
+      userName: ecUserName,
+      userId: ecUserUserId,
+      partyId: ecUserPartyId
+    }
+  }
```

Comment:
Else statement will now be triggered also when minimumSBLVersion(22, 5) is not met

Reply:
The test that uses ecUserData is also skipped when minimumSBLVersion(22, 5) is not met. Should I avoid setting ecUserData if it's not going to be used?

Reply:
The data is strictly for TT02 so would assume it's only set based on environment regardless of version. 
Good that it's handled now through the version check in the test using it, but some one else adding a test later might not know this and use the data.

Reply:
I replaced the `else `with an `else if (environment == 'tt02')` to make it more explicit.

Reply:
Would have been slightly cleaner if the if/else-if where the other way around